web_client/views/imageViewerWidget/geospatial.js

```diff
@@ -0,0 +1,83 @@
+import ImageViewerWidget from './base';
```

Comment:
Can you change this file to `import OpenlayersImageViewerWidget from './openlayers.js', and then extend that (which should allow you to remove all but the `render` method)?